src/sentry/lang/native/symbolicator.py

```diff
@@ -0,0 +1,138 @@
+from __future__ import absolute_import
+
+import six
+
+import time
+import logging
+
+from django.core.urlresolvers import reverse
+
+from requests.exceptions import RequestException
+
+from sentry import options
+from sentry.cache import default_cache
+from sentry.utils import metrics
+from sentry.net.http import Session
+from sentry.tasks.store import RetrySymbolication
+
+MAX_ATTEMPTS = 3
+REQUEST_CACHE_TIMEOUT = 3600
+SYMBOLICATOR_TIMEOUT = 5
+
+logger = logging.getLogger(__name__)
+
+
+def run_symbolicator(stacktraces, modules, project, arch, signal, request_id_cache_key):
+    self_url_prefix = options.get('system.url-prefix')
```

Comment:
This should use `system.internal-url-prefix` and fall back to `system.url-prefix` if not set.

Reply:
I'm also wondering if we should raise here with a better error message that would hint on premise users what to do in order to configure this "properly". We've had a couple of confused questions in the forum because of bad values in `sentry.url-prefix` that were picked up by the chunk upload endpoint.

Reply:
@jan-auer I can't tell where, but there are some validations on serverstartup for url-prefix

Reply:
Yeah, I think the problem is that we didn't strip the trailing slash in the beginning and `sentry-cli` was then issuing wrong requests. But since this is happening here, we shouldn't run into problems ™️